src/Platforms/AbstractPlatform.php

```diff
@@ -185,6 +185,19 @@ private function initializeAllDoctrineTypeMappings()
         }
     }
 
+    /**
+     * Returns the SQL snippet used to declare a column that can
+     * store characters in the ASCII character set
+     *
+     * @param mixed[] $column
+     *
+     * @return string
+     */
+    public function getAsciiTypeDeclarationSQL(array $column)
```

Comment:
Let's keep the naming consistent. It should be `getAsciiStringTypeDeclarationSQL()` (https://github.com/doctrine/dbal/pull/4274#discussion_r489842653).